### ISSUE TITLE
False-positive duplicate key error for aliased qualified key

### ISSUE THREAD

**Issue by drewinglis** [borkdude/clj-kondo#880]
**version**

```
$ clj-kondo 
clj-kondo v2020.05.02
```

**macro usage**

N/A

**platform**

OSX (installed via `brew install`)

**editor**

N/A

**problem**

`clj-kondo` gives false-positive duplicate key errors for aliased qualified keys if you also have the same non-aliased qualified key in the map.

**repro**

```
$ cat foo.clj 
(ns foo
  (:require [foo.bar :as bar]))


(def foo {:bar/id "asdf"
          ::bar/id "lkj"})
$ clj-kondo --lint foo.clj 
foo.clj:6:11: error: duplicate key ::bar/id
linting took 9ms, errors: 1, warnings: 0
```

**config**

N/A

**expected behavior**

It should treat the `::bar/id` key as `:foo.bar/id` and not give a duplicate key error.

**Comment by borkdude** [comment]
Thanks. Will fix soon.
